queries/java.scm

```diff
@@ -326,3 +326,43 @@
   body: (_) @value
   (#not-type? @value block)
 ) @_.domain
+
+;;!! foo(name: string) {}
+;;!      ^^^^^^^^^^^^
+(
+  (formal_parameters
+    (_)? @_.leading.endOf
+    .
+    (_) @argumentOrParameter
+    .
+    (_)? @_.trailing.startOf
+  ) @_dummy
+  (#not-type? @argumentOrParameter "block_comment")
+  (#single-or-multi-line-delimiter! @argumentOrParameter @_dummy ", " ",\n")
```

Comment:
won't this lose indentation?

Reply:
This is the same as we do in javascript. Our code takes care of indentation matching whenever you insert a new line.

Reply:
Well aren't we just fancy then 💅

Reply:
Oh yes :D